topics/imaging/tutorials/omero-suite/tutorial.md

```diff
@@ -0,0 +1,392 @@
+
+> <hands-on-title>Create the inputs for the OMERO pipeline</hands-on-title>
+>
+> 1. Create a new workflow in the workflow editor.
+>
+>    {% snippet faqs/galaxy/workflows_create_new.md %}
+>
+> 2. Select {% icon tool %} **Input dataset collection** from the list of tools:
+>    - {% icon param-collection %} **1: Input dataset collection** appears in your workflow. 
+>    Change the "Label" of this input to *Input image Dataset*.
+> 3. Add two {% icon tool %} **Input dataset**:
+>    - {% icon param-file %} **2: Input Dataset** and {% icon param-file %} **3: Input Dataset** appears in your workflow. 
+> Change the "Label" of these inputs to *Input Tabular ROIs* and *Input Metadata File*
+> 4. Add three {% icon tool %}**Simple Input for Workflow Logic**:
+>    - **4: Simple input for workflow logic**, **5: Simple input for workflow logic**, 
+>    **6: Simple input for workflow logic**  appear in your workflow. 
+>    Change the "Label" of these inputs to *Target Dataset Name*, *OMERO instance address* and
+>    *Annotation type*
+>    
+{: .hands_on}
+
+### Step 2 - OMERO Import
+We now add the step for the image import
+
+> <hands-on-title>Add the image upload step to the workflow</hands-on-title>
+>
+>1. While in the workflow editor add {% tool [OMERO Image Import](https://toolshed.g2.bx.psu.edu/view/ufz/omero_import/5.18.0+galaxy3) %}  from the list of tools:
+>   - Connect the output of {% icon param-file %} **1: Input image Datasets** to the "Images to Import in OMERO" 
+>   input of {% icon tool %} **7: OMERO Image Import**.
+>   - Connect the output of {% icon param-file %} **4: Target Dataset Name** to the "Target Dataset Name" 
+>   input of {% icon tool %} **7: OMERO Image Import**.
+>   - Connect the output of {% icon param-file %} **5: OMERO instance address** to the "OMERO host url"
+>   input of {% icon tool %} **7: OMERO Image Import**.
+{: .hands_on}
+
+### Step 3 - Dynamically parse image IDs 
+We need now a small series of steps to dynamically parse the IDs of the uploaded images.
+This can be done with the following tool sequence:
+
+> <hands-on-title>Parse Images IDs </hands-on-title>
+>
+>1. Add {% icon tool %} **Convert** from the list of tools:
+>   - Connect the output of {% icon tool %} **7: OMERO Image Import** to the {% icon tool %} **8: Convert** input
+>2. Add {% icon tool %} **Replace text**  from the list of tools:
+>   - Connect the output of {% icon tool %} **8: Convert** to the {% icon tool %} **Replace text** input 
+>3. Add {% icon tool %} **Split File**  from the list of tools:
+>   - Connect the output of {% icon tool %} **9: Replace text** to the {% icon tool %} **10: Split File** 
+>4. Add {% icon tool %} **Parse parameter value**  from the list of tools:
+>   - Connect the output of {% icon tool %} **10: Split File**  to the {% icon tool %} **11: Parse parameter value**
+{: .hands_on}
+
+This sequence of tools dynamically parses the single image ID which are use them for input of the metadata and 
```

Comment:
Can you please look at this sentence, I was not able to understand it properly to fix it.